tests/test_feature_manager_refresh.py

```diff
@@ -21,8 +21,13 @@ def test_refresh(self):
         feature_manager = FeatureManager(feature_flags)
         assert feature_manager is not None
         assert feature_manager.is_enabled("Alpha")
+        assert feature_manager.is_enabled("Alpha")  # test cache
```

Comment:
I don't think this proves the cache is working. Can we check if the cache object contains a record instead? 

Code coverage might prove it- but I'm unable to see that.